jazz_create-serverless-service/index.js

```diff
@@ -221,9 +223,30 @@ module.exports.handler = (event, context, cb) => {
             if (event.body.slack_channel) {
                 inputs.SLACKCHANNEL = event.body.slack_channel;
             }
-            if ((event.body.service_type === "api" || event.body.service_type === "function") && (event.body.require_internal_access !== null)) {
+
+            if ((event.body.service_type === "api" || event.body.service_type === "function") && (event.body.require_internal_access)) {
```

Comment:
you can extract event.body into a variable and re-use it in your code

Reply:
In cooperated the change